src/main/io/displayport_msp_bf_compat.c

```diff
@@ -293,34 +294,35 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_THR:
         return BF_SYM_THR;
 
-/*
     case SYM_TEMP_F:
-        return BF_SYM_TEMP_F;
+        return BF_SYM_TEMPERATURE;
 
     case SYM_TEMP_C:
-        return BF_SYM_TEMP_C;
-*/
+        return BF_SYM_TEMPERATURE;
+
     case SYM_BLANK:
         return BF_SYM_BLANK;
-/*
+
     case SYM_ON_H:
-        return BF_SYM_ON_H;
+        return BF_SYM_SPEED;
 
     case SYM_FLY_H:
-        return BF_SYM_FLY_H;
-*/
+        return BF_SYM_TOTAL_DISTANCE;
+
```

Comment:
Again, the above 2 changes are not consistent with what is actually being show.  This just adds confusion.